broker/broker_utils/requirements.txt

```diff
@@ -4,9 +4,13 @@ fastavro
 google-cloud-bigquery<2.10
 # apache-beam 2.25.0 requires pyarrow<0.18.0,>=0.15.1
 # but google-cloud-bigquery>=2.10.0 requires pyarrow>=3
+google-cloud-logging
 google-cloud-storage>=1.38
 google-cloud-pubsub<2  # beam requires <2
-numpy
+numpy==1.19.5
+# older pyarrow versions require numpy<1.20 to successfully convert datatypes
```

Comment:
Put comments before the code line, in this case `numpy=1.19.5`

Reply:
Do you know specifically which older pyarrow versions?

Reply:
I don't know, and I didn't see the answer anywhere in the official Issue. My best guess is pyarrow<1.0.0 (versions 1.0.0 to 5.0.0 were all developed in less than 1 year).